sigstore-java/build.gradle.kts

```diff
@@ -23,7 +23,10 @@ dependencies {
     implementation("dev.sigstore:protobuf-specs:0.1.0") {
         because("It generates Sigstore Bundle file")
     }
-    implementation("com.google.protobuf:protobuf-java-util:3.21.12") {
+    implementation(platform("com.google.protobuf:protobuf-bom:3.22.0")) {
+        because("We need consistent protobuf library versions")
```

Comment:
This doesn't feel necessary, we know what boms are for. It feels like it just clutters the build file.